superset/assets/src/explore/components/controls/DateFilterControl.jsx

```diff
@@ -494,7 +495,9 @@ export default class DateFilterControl extends React.Component {
   }
   render() {
     let value = this.props.value || defaultProps.value;
-    value = value.split(SEPARATOR).map((v, idx) => v.replace('T00:00:00', '') || (idx === 0 ? '-∞' : '∞')).join(SEPARATOR);
+    const endpoints = this.props.endpoints;
+    const iso8601 = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}$/;
```

Comment:
I'd pull this out to the top of the file and name is something like `ISO_8601_REGEX_MATCH`